spec/support/test_ledger/customer/operations/update.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module LedgerSync
+  module Ledgers
+    module TestLedger
+      class Customer
+        module Operations
+          class Update < TestLedger::Operation::FullUpdate
+            class Contract < LedgerSync::Ledgers::Contract
+              params do
+                required(:external_id).maybe(:string)
+                required(:ledger_id).filled(:string)
+                required(:PrimaryEmailAddr).maybe(:hash, Types::Reference)
+                optional(:DisplayName).filled(:string)
+                optional(:firstName).maybe(:string)
+                optional(:lastName).maybe(:string)
+                required(:PrimaryPhone).maybe(:hash, Types::Reference)
+                required(:subsidiary).maybe(:hash, Types::Reference)
```

Comment:
I'm pretty sure we dropped `subsidiary` from QBO as it doesnt exist there. Also why the difference in the camel case here?

Reply:
Updated, but these weren't even being used.